pkg/node/getBalanceOf.go

```diff
@@ -0,0 +1,31 @@
+package node
+
+import (
+	"fmt"
+	"strconv"
+)
+
+type GetBalanceOfResponse struct {
+	CandidateBalance float64
+	FinalBalance     float64
+}
```

Comment:
Too many useless information.
My suggestion:
```suggestion
type Balance struct {
	Candidate float64
	Final     float64
}
```